test/app/test/interactive.clj

```diff
@@ -1,8 +1,17 @@
 (ns app.test.interactive
   (:use [app.interactive]
+        [app.javaparser.navigation]
+        [app.operations]
+        [app.itemsOnLifecycle]
         [conjure.core]
         [clojure.test]))
 
+(def javaparser-cus-path
+  "test-resources/sample-codebases/javaparser/")
+
+(defn- command-sequence->input-str [command-sequence]
```

Comment:
It is the recommended way according to https://github.com/bbatsov/clojure-style-guide#arrow-instead-of-to